src/pkg/packager/deploy.go

```diff
@@ -444,6 +459,119 @@ func (p *Packager) pushImagesToRegistry(componentImages []string, noImgChecksum
 	}, 3, 5*time.Second)
 }
 
+func (p *Packager) pushReportsToRegistry(reportsPath string, name string) error {
+	message.Debug("deploy.pushReportsToRegistry()")
```

Comment:
this entire function will be re-written to use `oci.OrasRemote` after #1764 is merged